ctapipe/image/image_processor.py

```diff
@@ -209,51 +210,49 @@ def _parameterize_image(
         # parameterization
         return default
 
-    def _process_telescope_event(self, event):
+    def _process_telescope_event(self, tel_event):
```

Comment:
It's a `TelescopeEventContainer`, but yes, I'll add type hints for these. However, these are not *terribly* helpful, since at least my lsp only looks at the class variable, not the instance variables or rather assumes they are of the same type.